run.js

```diff
@@ -18,7 +18,8 @@ function run(command, environment) {
         console.log("Error running command:", error)
         reject(error);
       }
-      else if (stderr && stderr.length > 0) {
+      else if (stderr && stderr.length > 0 &&
+               !stderr.startsWith('Calling manager for ')) {
```

Comment:
The only comment I've got is to recommend that you don't hardcode this exception but instead put a '2> /dev/null' redirect on the Vaani.js commands that are known to write benign stuff to stderr. I think I had to do that with one of the existing commands I had in one of the files.